spec/requests/swagger_spec.rb

```diff
@@ -2922,13 +2922,13 @@
       end
 
       it 'supports the address validation api' do
-        address = build(:va_profile_address, :multiple_matches)
+        address = build(:va_profile_v3_validation_address, :multiple_matches)
```

Comment:
Do you want to support both the v3 address validation and the old address validation?